src/methods/method_overloading/hard/q10/Overload10.java

```diff
@@ -0,0 +1,59 @@
+package methods.method_overloading.hard.q10;
+
+/*
+Write a Java Program for a hospital that displays the name and the gender of the baby to the parents. The method takes
```

Comment:
Change to:
Write a Java Program for a hospital that displays the name and the gender of a baby to the parents. The main method should take only the name or both the name and a boolean for the gender (true = girl, false = boy). The program should pass the input information to an overloaded method called baby which takes either a String or a String and a boolean, and returns a String. The main method should then print out the returned String. (Hint: You may want to use the Scanner method .hasNextBoolean)